modules/javafx.controls/src/main/java/javafx/scene/control/skin/TableColumnHeader.java

```diff
@@ -372,8 +378,9 @@ public final ReadOnlyObjectProperty<TableColumnBase<?,?>> tableColumnProperty()
             isSizeDirty = false;
         }
 
+        double cornerRegionPadding = tableHeaderRow == null ? 0d : tableHeaderRow.cornerPadding.get();
```

Comment:
minor: could we use consistent 0 or 0.0 or 0d?
(I'd suggest 0.0 to avoid int->double conversion, or 0d as very few people use 0d)

Reply:
Indeed it is not consistent at all throughout all the source code...

> or 0d as very few people use 0d

do you mean `or 0 as very few people use 0d` or `or 0d as very few people use 0`? 

Checking Controls source code, doubles use `0` or `0d` or `0.0d` as well as 0.0, so it will be hard to be consistent without making too many changes... For instance, next line is `double sortWidth = 0;`. If I go to 0.0, should I change it as well?